sources/packages/backend/workflow/test/test-utils/factories/assessment-consolidated-data.ts

```diff
@@ -283,3 +284,57 @@ export function createParentsData(options?: {
   }
   return parentsData;
 }
+
+/**
+ * Provides the necessary data to the assessment workflow, defining the student as dependant, and
+ * create the parents data to be used in the assessment, for instance, if they can report using
+ * the supporting users portal or if the student will report the parents' information on their behalf.
+ * @param options creation options.
+ * - `dataType` indicates which moment the assessment data should represents.
+ * - `parents` parents data to be used in the assessment. If not provided, a default parent will be created.
+ * - `numberOfParents` generate information for one or two parents. If provided, it will override the `parents` option.
+ * @returns parents data to be used.
+ */
+export function createIdentifiableParentsData(options?: {
+  dataType?: AssessmentDataType;
+  parents?: IdentifiableParentData[];
+  numberOfParents?: 1 | 2;
+}): Partial<AssessmentConsolidatedData> {
+  // Default values for options when not provided.
+  const dataType = options.dataType ?? AssessmentDataType.Submit;
+  // Make the student a dependant.
+  const parentsData = {} as Partial<AssessmentConsolidatedData>;
+  parentsData.studentDataDependantstatus = "dependant";
+  if (options?.numberOfParents) {
+    parentsData.studentDataParents = Array.from(
+      { length: options.numberOfParents },
+      (_, i) => ({
+        parentFullName: `Parent ${i + 1}`,
+        parentIsAbleToReport: YesNoOptions.Yes,
+      }),
+    );
+  } else {
+    parentsData.studentDataParents = options?.parents ?? [
+      {
+        parentFullName: "Parent 1",
```

Comment:
is `parentFullName` required for E2E test as it will not be present in workflow data scope.

Reply:
True, let me review it.

Reply:
Removed.